pkg/template/processor.go

```diff
@@ -77,6 +77,7 @@ func (p Processor) Apply(objs []runtime.RawExtension) error {
 }
 
 func createOrUpdateObj(cl client.Client, obj runtime.Object) error {
+	fmt.Printf("creating or updating %v\n", obj)
```

Comment:
Remove Printf. You can log it properly if needed though :)

Reply:
doh! (and done in 198b954)